package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-responsive": "^10.0.0"
```

Comment:
What is the reason why we installed this? I think MUI is capable of updating the responsiveness.

Reply:
I used this initially but saw that MUI already have this function.